single-audit/single_audit/settings/base.py

```diff
@@ -41,7 +32,6 @@
     'localflavor',
     'fac.apps.FacConfig',
     'distiller.apps.DistillerConfig',
-    'resolve_findings.apps.ResolveFindingsConfig',
```

Comment:
@alexsoble This is an artifact of the work-in-progress on #79; thanks for catching!

Reply:
👍 👍